### ISSUE TITLE
Table filter looses focus in a tabbed interface

### ISSUE THREAD

**Issue by MaximMartel** [wet-boew/wet-boew#8019]
Hi, 

When you put a table with filtering inside a tabbed interface, the filter looses focus when you type in something in the field. 

Please see working example : https://maximmartel.github.io/WET-BOEW-TestTabbedInterface/unmin/demos/tabs/tabs-en.html

You cant type a word without clicking back in the text area after pressing on a letter or number.

**Comment by kodecount** [comment]
All of these issues are related (THERE IS MANY MORE):
https://github.com/wet-boew/wet-boew/issues/7996
https://github.com/wet-boew/wet-boew/issues/7949
https://github.com/wet-boew/wet-boew/issues/7820
https://github.com/wet-boew/wet-boew/issues/7475
https://github.com/wet-boew/wet-boew/issues/7472
https://github.com/wet-boew/wet-boew/issues/7385
https://github.com/wet-boew/wet-boew/issues/5720

Common Thread:
https://github.com/wet-boew/wet-boew/issues/7752

**Comment by MaximMartel** [comment]
@duboisp

**Comment by EricDunsworth** [comment]
@kodecount I don't think all of those are related to what @MaximMartel described in this issue's OP:
* #5720 was a proposed table filter enhancement that was declined and closed 3 years ago (since it had the potential of confusing certain users).
* #7752 is requesting the removal of a broken equalization feature in the tabbed interface plugin, but doesn't identify it as the cause of any focusing issues related to filter table implementations nested within tabbed interfaces.

If this issue is in fact a duplicate of #7472/#7475, I should note that @josegar74 opened PR #7581 to fix it. That PR is still open to this day.

@lucas-hay @duboisp Since several issues describe this problem, IMO there would be value in seeking out the very first issue that identified it and closing out the other issues as duplicates (so long as we're completely sure they are in fact duplicates - as opposed to unrelated focusing problems).

**Comment by lucas-hay** [comment]
Thanks @EricDunsworth, we will discuss in OR review meeting this week.

**Comment by duboisp** [comment]
@EricDunsworth are you going to try to fix this table filtering "focus" related issue? 

@EricDunsworth Are you 110% sure the other issue that you potentially identified as "duplicate" is a real duplicate? If you doubt keep them open for now.

**Comment by EricDunsworth** [comment]
@duboisp 

> @EricDunsworth are you going to try to fix this table filtering "focus" related issue?

No. But as I alluded to in my previous comment, PR #7581 might potentially fix it.

> @EricDunsworth Are you 110% sure the other issue that you potentially identified as "duplicate" is a real duplicate? If you doubt keep them open for now.

I don't know for sure whether #7472/#7475 are duplicates of this issue (hence the unsure tone I used).

All I did was glance at the issues @kodecount referenced, identified some that may be out of scope and the fact that PR #7581 (which is still open to this day) is intended as a fix for #7472. If this issue is a duplicate of #7472, then that means a fix/workaround may already exist for it.

IMO all of the issues referenced in @kodecount's comment (except for #5720) are are related. Some are exact duplicates of this one and others are similar. I was mistaken about #7752 being unrelated in my previous comment. Upon further research, I think it is related. The tabbed interface's broken equal heights functionality seems to be causing the focusing problems covered throughout these issues.

Btw I'm only a WET contributor (not a maintainer), so I lack the capability of closing/labeling issues and PRs.

**Comment by toasinger** [comment]
Hey guys, I found a more consistent workaround for this.

$(document).on('keypress', $('input[type="search"]'), function() {
 $(".wb-tabs").removeClass("wb-eqht wb-eqht-inited");
 $('.tabpanels').css("min-height", 0); //Hack out that min-height being added dynamically out to the tab panel to keep any footers, etc moored to the bottom of the table. (eliminates big whitespace.)
 });

That finds the datatable search input box... and while typing in there is applying the work around consistently. On tab change, size change, etc... also removes that annoying min-height.

**Comment by baohongw** [comment]
@toasinger, when I added your solution inside <script> to html file, my table Filter disappeared, show 1 of 10 and similar title disappeared. What's exactly location to put this code? Did anyone encounter the new issue after this fixing?
